packages/perl.rb

```diff
@@ -3,23 +3,23 @@
 class Perl < Package
   description 'Perl 5 is a highly capable, feature-rich programming language with over 29 years of development.'
   homepage 'https://www.perl.org/'
-  version '5.32.0'
+  version '5.32.1'
   license 'GPL-1+ or Artistic'
   compatibility 'all'
-  source_url 'http://www.cpan.org/src/5.0/perl-5.32.0.tar.gz'
-  source_sha256 'efeb1ce1f10824190ad1cadbcccf6fdb8a5d37007d0100d2d9ae5f2b5900c0b4'
+  source_url 'http://www.cpan.org/src/5.0/perl-5.32.1.tar.gz'
+  source_sha256 '03b693901cd8ae807231b1787798cf1f2e0b8a56218d07b7da44f784a7caeb2c'
 
-  binary_url ({
-    aarch64: 'https://downloads.sourceforge.net/project/chromebrew/armv7l/perl-5.32.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://downloads.sourceforge.net/project/chromebrew/armv7l/perl-5.32.0-chromeos-armv7l.tar.xz',
-       i686: 'https://downloads.sourceforge.net/project/chromebrew/i686/perl-5.32.0-chromeos-i686.tar.xz',
-     x86_64: 'https://downloads.sourceforge.net/project/chromebrew/x86_64/perl-5.32.0-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://downloads.sourceforge.net/project/chromebrew/perl-5.32.1-chromeos-armv7l.tar.xz',
```

Comment:
Shouldn't the aarch64 and armv7l downloads be identical?

Reply:
Fixed. Not sure what happened there. Looks like an errant upload.